From: Martin Botka <martin.botka@somainline.org>
To: Mark Rutland <mark.rutland@arm.com>,
	 Lorenzo Pieralisi <lpieralisi@kernel.org>,
	 Sudeep Holla <sudeep.holla@arm.com>,
	 "Rafael J. Wysocki" <rafael@kernel.org>,
	 Viresh Kumar <viresh.kumar@linaro.org>,
	Yangtao Li <tiny.windzz@gmail.com>,  Chen-Yu Tsai <wens@csie.org>,
	Jernej Skrabec <jernej.skrabec@gmail.com>,
	 Samuel Holland <samuel@sholland.org>,
	Rob Herring <robh+dt@kernel.org>,
	 Krzysztof Kozlowski <krzysztof.kozlowski+dt@linaro.org>,
	 Conor Dooley <conor+dt@kernel.org>
Cc: linux-arm-kernel@lists.infradead.org,
	linux-kernel@vger.kernel.org,  linux-pm@vger.kernel.org,
	linux-sunxi@lists.linux.dev,  devicetree@vger.kernel.org,
	Andre Przywara <andre.przywara@arm.com>,
	 Alan Ma <tech@biqu3d.com>,
	Luke Harrison <bttuniversity@biqu3d.com>,
	 Marijn Suijten <marijn.suijten@somainline.org>,
	 AngeloGioacchino Del Regno
	<angelogioacchino.delregno@somainline.org>,
	 Konrad Dybcio <konrad.dybcio@somainline.org>,
	 Rogerio Goncalves <rogerlz@gmail.com>,
	Martin Botka <martin@biqu3d.com>,
	 Martin Botka <martin.botka@somainline.org>
Subject: [PATCH 6/6] arm64: dts: allwinner: h616: Add cooling cells
Date: Mon, 04 Sep 2023 17:57:06 +0200	[thread overview]
Message-ID: <20230904-cpufreq-h616-v1-6-b8842e525c43@somainline.org> (raw)
In-Reply-To: <20230904-cpufreq-h616-v1-0-b8842e525c43@somainline.org>

Add cooling cells so we enable passive cooling on CPU by regulating
CPU voltage and frequency

Signed-off-by: Martin Botka <martin.botka@somainline.org>
---
 arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h616.dtsi
@@ -26,6 +26,7 @@
 			reg = <0>;
 			enable-method = "psci";
 			clocks = <&ccu CLK_CPUX>;
+			#cooling-cells = <2>;
 		};
 
 		cpu1: cpu@1 {
@@ -34,6 +35,7 @@
 			reg = <1>;
 			enable-method = "psci";
 			clocks = <&ccu CLK_CPUX>;
+			#cooling-cells = <2>;
 		};
 
 		cpu2: cpu@2 {
@@ -42,6 +44,7 @@
 			reg = <2>;
 			enable-method = "psci";
 			clocks = <&ccu CLK_CPUX>;
+			#cooling-cells = <2>;
 		};
 
 		cpu3: cpu@3 {
@@ -50,6 +53,7 @@
 			reg = <3>;
 			enable-method = "psci";
 			clocks = <&ccu CLK_CPUX>;
+			#cooling-cells = <2>;
 		};
 	};
 
